src/main/cpp/Robot.cpp

```diff
@@ -51,8 +51,11 @@ void Robot::TeleopInit() {
     // this line or comment it out.
     if (m_autonomousCommand != nullptr) {
         m_autonomousCommand->Cancel();
-        m_autonomousCommand = nullptr;
     }
+
+    teleop_command_ = m_container.GetDriveCommand();
+    VOKC_CALL(teleop_command_ != nullptr);
+    teleop_command_->Schedule();
```

Comment:
should we make the drive command the default command of the subsystem? so that way we don't have to worry about ever explicitly scheduling it?

Reply:
Probably, but this is how we implemented it last year. We should definitely improve this in the SwerveDrive